//Define the used version of proto
syntax = "proto3";

message Empty {
}

message Response {
    string train_set = 1;
    string valid_set = 2;
}


//Define the service
service RebaseDataset {
    rpc LoadData(Empty) returns (Response);
}
